packages/scandipwa/src/component/FieldSelect/FieldSelect.component.js

```diff
@@ -36,7 +36,12 @@ export class FieldSelect extends PureComponent {
         handleSelectExpand: PropTypes.func.isRequired,
         isDisabled: PropTypes.bool.isRequired,
         isUpDirection: PropTypes.bool.isRequired,
-        isScrollable: PropTypes.bool.isRequired
+        isScrollable: PropTypes.bool.isRequired,
+        isSortSelect: PropTypes.bool
+    };
+
+    static defaultProps = {
+        isSortSelect: false
     };
```

Comment:
So isn't always passed from field component? It is most lucky always will be with default value.